### ISSUE TITLE
Does mc-loss need more training epoches?

### ISSUE THREAD

**Issue by gbstack** [PRIS-CV/Mutual-Channel-Loss#16]
Hi,
I added mc-loss to mnasnet (https://arxiv.org/abs/1807.11626v3) network and train it on a custom fine-grained dataset. 
The total epoches is 15, initial LR is 2e-2, and final LR is 1e-5 using cosine LR scheduler.
But the validation accuracy at epoch 15 is 0.62, while the original mnasnet implementation reaches 0.78 val accuracy at epoch 15. 

The training epoches in the paper is 300, is this the cause?
(my GPU is slow, so I want to experiment for less epoches to determine whether mc-loss performs good on this dataset)

Thanks for your great work!

**Comment by dongliangchang** [comment]
Fine-tune or trained form scratch？
Hyper-parameter？
15 epoch is too small.

**Comment by gbstack** [comment]
> 
> 
> Fine-tune or trained form scratch？
> Hyper-parameter？
> 15 epoch is too small.

It's fine-tune using ImageNet pretrained weights (load pretrained parameters except the fc layer and last conv layer which is num_classes*3 channels and 1x1 filter size)

following is hyper parameters:
batch_size: 16
epochs: 15
infer_resize: 512
label_smooth: false
lr: 0.02
mc_loss: true
name: mnasnet_1.0
num_classes: 130
warmup_iters_ratio: 0.01
weight_decay: 1.0e-05
momentum: 0.9

optimizer is using SGD with momentum.

Thanks!

**Comment by dongliangchang** [comment]
The input of the MC-Loss should be the features before GAP (H>1 and W>1).
Maybe a bigger max_epoch will helpful.

**Comment by dongliangchang** [comment]
https://github.com/Kurumi233/Mutual-Channel-Loss/issues/5  
you can also ref this issue.
